diff --git a/node_modules/cac/dist/index.mjs b/node_modules/cac/dist/index.mjs
index 958a6e8..8d0aa7b 100644
--- a/node_modules/cac/dist/index.mjs
+++ b/node_modules/cac/dist/index.mjs
@@ -609,15 +609,6 @@ class CAC extends EventEmitter {
 }
 
 const cac = (name = "") => new CAC(name);
-if (typeof module !== "undefined") {
-  module.exports = cac;
-  Object.assign(module.exports, {
-    default: cac,
-    cac,
-    CAC: CAC,
-    Command: Command
-  });
-}
 
 export default cac;
 export { CAC, Command, cac };
